OrbitGl/Track.cpp

```diff
@@ -141,17 +142,19 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode) {
 
   // Draw collapsing triangle.
   float button_offset = layout.GetCollapseButtonOffset();
-  Vec2 toggle_pos = Vec2(tab_x0 + button_offset, m_Pos[1] + half_label_height);
+  float toggle_y_pos = m_Pos[1] + half_label_height;
+  Vec2 toggle_pos = Vec2(tab_x0 + button_offset, toggle_y_pos);
   collapse_toggle_->SetPos(toggle_pos);
   collapse_toggle_->Draw(canvas, picking_mode);
 
   if (!picking) {
     // Draw label.
     float label_offset_x = layout.GetTrackLabelOffsetX();
-    float label_offset_y = layout.GetTrackLabelOffsetY();
+    // Vertical offset for the text to be aligned to the center of the triangle.
+    float label_offset_y = GParams.font_size / 3.5;
```

Comment:
This 3.5 value seems arbitrary. As discussed, this doesn't seem work properly when changing the font size.

Reply:
After our talk, I started to look why this 3.5 doesn't work for you, and realized that I wasn't getting the updated font_size if you modify directly in the menu as you were making.

So, I made a GetFontSize() in Timegraph and now it is always updated. BTW, I modify this 3.5 to 3, no theorical explanation for that. With 3, the triangle is aligned with upper-case letters, and with 3.5 the triangle is aligned with lower-cases ones. As 3 in more elegant, I leaved 3 there.